### DISCUSSION TITLE
希望增加图片灯箱

### DISCUSSION THREAD

**Discussion by adkinsmpage** [YunYouJun/hexo-theme-starry#161] [Ideas 奇思妙想]
希望增加图片灯箱，类似 FancyBox 的那种功能，我自己已经尝试若干次，最后的结果要么是Error，要么是无法显示，期待作者能新增这种功能

**Comment by YunYouJun** [comment]
事实上该功能以前添加过，但是 FancyBox 依赖于 JQuery，两者相加后的包体积实际是相当大的，所以权衡后决定不添加 FancyBox。

所指是否有其他使用场景？

如果您指的是大量图片的这种情况，可以考虑建立单独的相册页。

可参见 https://yun.yunyoujun.cn/guide/page.html#%E7%9B%B8%E5%86%8C-albums

示例：https://www.yunyoujun.cn/albums/

> 使用更轻量的 lightgallery.js 实现

**Comment by adkinsmpage** [comment]
主要是文章里图片的实现。有些图片过大，被页面缩放，从而变得清晰度较低。

**Reply by adkinsmpage** [reply]
我试过很多无JQ方案，例如 KicoStyle，Medium-Zoom，LightBox等，但都无法正常使用，非常奇怪

**Reply by YunYouJun** [reply]
右键打开图片？:roft:
